Lib/ufo2ft/featureCompiler.py

```diff
@@ -22,6 +23,7 @@
 )
 
 logger = logging.getLogger(__name__)
+timer = Timer(logging.getLogger("fontmake.timer"), level=logging.INFO)
```

Comment:
let's stick to use one library-wide timer logger called in this case "ufo2ft.timer"